buildSrc/subprojects/buildquality/src/main/kotlin/gradlebuild.ci-reporting.gradle.kts

```diff
@@ -100,7 +100,7 @@ fun prepareReportsForCiPublishing(failedTasks: List<Task>, executedTasks: List<T
 
 fun Project.tmpTestFiles() =
     layout.buildDirectory.dir("tmp/test files").get().asFile.listFiles()?.filter {
-        Files.walk(it.toPath()).use { paths -> !paths.allMatch(Files::isDirectory) }
+        Files.walk(it.toPath()).use { paths -> !paths.allMatch { it.toFile().isDirectory } }
```

Comment:
These cases where disambiguation was necessary are caused by disabling the compiler progressive mode. It is mandatory to do so because it is only available if you target `apiVersion` and `languageVersion` of the same version as the compiler.